### PR TITLE
Add anyqt recipe

### PR DESCRIPTION
**Pull request by ales-erjavec**

### CODE REVIEW

File: recipes/anyqt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+  script:
+    - pip install --no-deps --no-index --no-cache ./
```
Comment (synapticarbors) [inline recipes/anyqt/meta.yaml]: Is there a strong reason for installing via pip instead of `python setup.py`?

Comment (ales-erjavec) [inline recipes/anyqt/meta.yaml]: pip includes a list of installed files in the generated egg-info directory (the `--record record.txt` usually lost after the standard setup.py install invocation).

I can remove it if its is a problem.

Comment (synapticarbors) [inline recipes/anyqt/meta.yaml]: I'm not completely familiar with the issue, but picking a random package from conda-forge and looking at the files it includes, I see:

- `info/files`
-  `lib/python3.5/site-packages/XXX.egg-info/SOURCES.txt`

both which contain lists of files in the package (The files aren't identical). Is the `SOURCES.txt` the file you're looking for?

Comment (ales-erjavec) [inline recipes/anyqt/meta.yaml]: > Is the SOURCES.txt the file you're looking for?

No. I am thinking of XXX.egg-info/installed-files.txt.
SOURCES.txt is a setuptools egg_info command byproduct. AFAIK it lists files for inclusion in an 'sdist', while installed-files.txt lists all files that are installed by the install command (relative to the egg-info directory)

Comment (ales-erjavec) [inline recipes/anyqt/meta.yaml]: I removed the use of pip. Now using the standard `setup.py install --single-version...`

File: recipes/anyqt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```
Comment (ales-erjavec) [inline recipes/anyqt/meta.yaml]: Fixed

**Pull request merged**